System/Cryptography/Encryption.py

```diff
@@ -8,37 +8,37 @@
+	    	# Generate Hash Key Using SHA256 #
+	    	Key = SHA256.new()
+	    	Key.update(bytes(SecretBytes))
+	   	 SecretKey = Key.hexdigest()
+	    	return SecretKey
 
-def GenerateSecret(PrivateKey, PublicKey): # Generates A New Secret #
-    Secret = pow(int(PublicKey), int(PrivateKey), Prime)
-    SecretBytes = str(Secret)
-	
-    # Generate Hash Key Using SHA256 #
-    Key = SHA256.new()
-    Key.update(bytes(SecretBytes))
-    SecretKey = Key.hexdigest()
-    return SecretKey
+	def Encrypt(self, Logger:object, Command:str):
 
-def Encrypt(Logger:object, Command:str):
-    
-    # Load The Pub/Private Keys #
-    PublicKey, PrivateKey = ReadKeys(Logger)
-    SecretKey = GenerateSecret(PublicKey, PrivateKey)
-    enc_Command= ""
+	    	# Load The Pub/Private Keys #
+	    	PublicKey, PrivateKey = ReadKeys(Logger)
+	    	SecretKey = GenerateSecret(PublicKey, PrivateKey)
+	    	enc_Command= ""
 
-    for c in Command:
-        enc_Command += chr(ord(c)+ SecretKey)
+	    	for c in Command:
+			enc_Command += chr(ord(c)+ SecretKey)

```

Comment:
@PRISHIta123 is this the part that deals with the encryption? if so, can we switch it out for something like AES256? Also, I just realized I may have been mistaken with forcing us to generate a prime for every time, I'm not familiar with DH key exchanges. I do know however that if you can it's better to offload the task of making potentially dangerous functions to other libraries wherever possible.

Reply:
Actually, if it doesn't matter what algorithm is used for the actual encryption, I can handle the encryption/decryption if you can handle the key-exchange.

Reply:
I was trying to use the hashlib library to generate the key but as you reported that it cannot be installed on the development server, I shifted to the SHA256 hash function of the pycrypto library. Even if we decide to use the AES algorithm as a replacement for Diffie-Hellman, the key generation will still be required. 

If the issue with the code is something related to the key exchange, maybe I can try fixing that, but I am not sure if changing the algorithm entirely will resolve it. Perhaps it would help me if you could specify where exactly the problem is arising.